includes/HookHandlers/Main.php

```
@@ -0,0 +1,373 @@
+		Parser $parser,
+		string $text = '',
+		string $heading = ' ',
+		int $maxLength = -1,
+		string $page = '?page?',
+		string $link = 'default',
+		bool $trim = false
+	): string {
+		$parser->addTrackingCategory( 'dplchapter-parserfunc-tracking-category' );
+		$output = LST::extractHeadingFromText(
+			$parser, $page, $text, $heading, '',
+			$sectionHeading, true, $maxLength, $link, $trim
```

PHPMD (2.15.0)

Avoid unused local variables such as '$sectionHeading'. (Unused Code Rules)

(UnusedLocalVariable)